pkg/monitortests/monitoring/disruptionmetricsapi/monitortest.go

```diff
@@ -70,10 +70,10 @@ func (w *availability) StartCollection(ctx context.Context, adminRESTConfig *res
 		return w.notSupportedReason
 	}
 
-	disruptionBackedName := "metrics-api"
-	newConnectionTestName := fmt.Sprintf("[sig-instrumentation] disruption/%s connection/new should be available throughout the test", disruptionBackedName)
-	reusedConnectionTestName := fmt.Sprintf("[sig-instrumentation] disruption/%s connection/reused should be available throughout the test", disruptionBackedName)
-	url := "/apis/metrics.k8s.io/v1beta1"
+	disruptionBackedName := "kube-api"
```

Comment:
I'm thinking you handle this in [pkg/monitortests/kubeapiserver/disruptionexternalapiserver/monitortest.go](https://github.com/openshift/origin/pull/29058/files#diff-2c72988a21dc176d8f5753c318ec78e43efddac833c7f82fca57bed9bc3d3517R47) and don't really want to remove the metrics-api backends?